webapp/locales/en.json

```diff
@@ -294,12 +294,14 @@
       },
       "columns": {
         "name": "Name",
-        "slug": "Slug"
+        "slug": "Slug",
+        "unblk": "Unblock"
       },
       "empty": "So far, you have not blocked anybody.",
       "how-to": "You can block other users on their profile page via the content menu.",
       "block": "Block user",
-      "unblock": "Unblock user"
+      "unblock": "Unblock user",
+      "unblocked": " is unblocked again"      
```

Comment:
I don't think we need (or should have) the `space` here – you're adding one in the component anyway! Same goes for the German translation.

Reply:
ok